openedx/core/djangoapps/programs/tests/test_models.py

```diff
@@ -1,7 +1,7 @@
 """
 Tests for models supporting Program-related functionality.
 """
-
+import ddt
 from mock import patch
 
 from django.test import TestCase
```

Comment:
Minor: ddt, mock, and Django are all third-party packages; they should be together in a single group of imports.